deepchecks/nlp/checks/data_integrity/under_annotated_segments.py

```diff
@@ -232,6 +249,8 @@ class UnderAnnotatedPropertySegments(UnderAnnotatedSegments):
         Maximum number of samples to use for this check.
     n_to_show : int , default: 3
         number of segments with the weakest performance to show.
+    annotation_ratio_threshold : float , default: 95.0
```

Comment:
I think we should not make it a argument but a const in the class

Reply:
Done